src/Reader/AbstractFeed.php

```diff
@@ -91,8 +91,10 @@ public function __construct(DOMDocument $domDocument, $type = null)
      * a self-referencing URI.
      *
      * @param string $uri
+     *
+     * @return void
      */
-    public function setOriginalSourceUri($uri)
+    public function setOriginalSourceUri($uri): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.